doc/zettel.txt

```diff
@@ -288,6 +289,9 @@ option to other value than 0.
 
 The `template` field is discussed in |Vim-Zettel-Templates|.
 
+The `rel_path` field determines where new notes will be created, relative to
```

Comment:
I would add a simple example here, it is not apparent to me what form of path it should be?

- .zettel/
- zettel
- /zettel

etc etc

Reply:
Thanks, done, it should be `zettel/`. It's also in the example configuration.